addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -130,3 +189,66 @@ def test_edi_json(self):
                 "OthChrg": 0.0, "TotItemVal": 0.0}],
         })
         self.assertDictEqual(json_value, expected, "Indian EDI with 0(zero) quantity sent json value is not matched")
+
+        #=================================== Negative unit price test =============================================
```

Comment:
why did not we put it in different methods?  We could at least do it for these.

Reply:
because use expected with update

Reply:
what I need I update in expected

Reply:
It is still a bit confusing to follow to have update on update, but I suppose we can update later as well. (better to have expected reset to its original value)